indico/modules/events/registration/util_test.py

```diff
@@ -109,3 +111,53 @@ def test_import_error(dummy_regform):
         import_registrations_from_csv(dummy_regform, BytesIO(csv))
     assert 'missing first' in str(e.value)
     assert 'Row 2' in str(e.value)
+
+
+@pytest.mark.parametrize(('start_dt', 'end_dt', 'include_scheduled', 'expected_regform_flag'), (
+    (datetime(2007, 1, 1, 1, 0, 0), datetime(2007, 2, 1, 1, 0, 0), False, False),
+    (datetime(2019, 1, 1, 1, 0, 0), datetime(2020, 2, 1, 1, 0, 0), False, True),
+    (datetime(2007, 1, 1, 1, 0, 0), datetime(2007, 2, 1, 1, 0, 0), True, True),
+    (datetime(2019, 1, 1, 1, 0, 0), datetime(2020, 2, 1, 1, 0, 0), True, True),
+    (None, datetime(2020, 2, 1, 1, 0, 0), False, False),
+    (None, datetime(2020, 2, 1, 1, 0, 0), True, False),
+    (datetime(2019, 1, 1, 1, 0, 0), None, False, True),
+    (None, None, False, False),
+    (None, None, True, False)
+))
+def test_get_event_regforms_no_registration(dummy_event, dummy_user, dummy_regform, freeze_time, start_dt, end_dt,
+                                            include_scheduled, expected_regform_flag):
+    freeze_time(datetime(2019, 12, 13, 8, 0, 0))
+    if start_dt:
+        dummy_regform.start_dt = dummy_event.tzinfo.localize(start_dt)
+    if end_dt:
+        dummy_regform.end_dt = dummy_event.tzinfo.localize(end_dt)
+
+    regforms, registrations = get_event_regforms_registrations(dummy_event, dummy_user, include_scheduled)
+
+    assert ('Registration Form' in str(regforms)) == expected_regform_flag
+    assert 'None' in str(registrations.values())
```

Comment:
same here, `assert None in registrations.values()`

or if you know there's exactly one entry, just check `registrations.values() == [None]`